packages/lib/server-side/src/utils.ts

```diff
@@ -16,7 +16,7 @@ export async function auth(
     res: Response,
     next: NextFunction,
     ensName: string,
-    db: IAccountDatabase,
+    doesAccountExist: (ensName: string) => Promise<boolean>,
```

Comment:
A name like hasAccount would be more concise. getAccount, setAccount, hasAccount

Reply:
good idea!

Reply:
done